utilities/test_suite/HIP/runImageTests.py

```diff
@@ -63,7 +63,7 @@ def run_unit_test(srcPath1, srcPath2, dstPathTemp, case, numRuns, testType, layo
             if layout == 2 and outputFormatToggle == 1:
                 continue
 
-            if case == "40" or case == "41" or case == "49" or case == "54":
+            if case == "40" or case == "41" or case == "49" or case == "51" or case == "54":
```

Comment:
Could you issue a quick separate PR to fix this across all RPP python files?

We need enums with the same names like the user can provide to run tests with the script - Like "BOX_FILTER", "MEDIAN_FILTER".